### ISSUE TITLE
BUG: The initial toolbar move button does not get removed if you set the draggable prop to false programmatically

### ISSUE THREAD

**Issue by am1rb** [artf/grapesjs#4255]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chrome v98.0.4758.80

### Reproducible demo link

https://jsfiddle.net/am1rb/3uzkn5t7/8/

### Describe the bug

**How to reproduce the bug?**
1. Render a simple component on the canvas
2. Set draggable=false for the component programmatically
3. Select the component on the editor
4. The move button exists on the tooltip but it does not work

**What is the expected behavior?**
The component toolbar should be rendered and the move button is supposed to get removed if the draggable prop is set to false. There is the same issue with copyable and removable props as well.

**What is the current behavior?**
The move button exists but it does not work and the below error appears on the console:
![image](https://user-images.githubusercontent.com/573937/163155247-6a20c81b-13a1-4e1e-820b-5ca023c76c98.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by artf** [comment]
Yeah the toolbar is not really dynamic, so if you need to recalculate it, you have to force it manually
```js
model.set({ draggable: false, toolbar: null })
model.initToolbar();
```
